experiment/measurer/test_run_crashes.py

```diff
@@ -61,3 +61,15 @@ def test_filter_crash_type():
     assert (run_crashes._filter_crash_type('Heap-buffer-overflow\nWRITE {*}') ==
             'Heap-buffer-overflow\nWRITE')
     assert run_crashes._filter_crash_type('Timeout') == 'Timeout'
+
+
+# pylint: disable=protected-access
+def test_filter_crash_state():
+    """Tests _filter_crash_state."""
+    assert (run_crashes._filter_crash_state(
+        'Segv on unknown address:int* std::__1::fill_n<int*, long, int>\n'
+        'int arrow::util::RleDecoder::GetBatchWithDict<int>\n'
+        'parquet::DictDecoderImpl<parquet::PhysicalType<\n') ==
+            'Segv on unknown address:int* std::__1::fill_n\n'
+            'int arrow::util::RleDecoder::GetBatchWithDict\n'
+            'parquet::DictDecoderImpl<parquet::PhysicalType<\n')
```

Comment:
Should we try to handle this case as well? We're in a bit of a weird state either way with the truncated lines, so I don't think too much could go wrong by removing anything after < to the end either.

Reply:
Done, added to regex for truncated stuff.